src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/edit/files/main.js

```diff
@@ -63,235 +38,154 @@ define(function() {
          */
-        editMedia: function(additionalMedia) {
-            this.sandbox.emit('husky.datagrid.items.get-selected', function(selected) {
-                this.sandbox.emit('husky.dropzone.' + this.options.instanceName + '.lock-popup');
-                // add additional media to the edit-list, but only if its not already contained
-                if (!!additionalMedia && selected.indexOf(additionalMedia) === -1) {
-                    selected.push(additionalMedia);
-                }
-                this.sandbox.emit('sulu.media.collections.edit-media', selected);
+        bindListToolbarEvents: function() {
+            // show dropzone popup
+            this.sandbox.on('sulu.list-toolbar.add', function() {
+                this.sandbox.emit('husky.dropzone.' + this.options.instanceName + '.show-popup');
             }.bind(this));
-        },
 
-        /**
-         * Starts the dropzone component
-         */
-        startDropzone: function() {
-            this.sandbox.start([
-                {
-                    name: 'dropzone@husky',
-                    options: {
-                        el: this.$find(constants.dropzoneSelector),
-                        url: '/admin/api/media?collection=' + this.options.data.id,
-                        method: 'POST',
-                        paramName: 'fileVersion',
-                        instanceName: this.options.instanceName
-                    }
-                }
-            ]);
-        },
+            // delete a media
+            this.sandbox.on('sulu.list-toolbar.delete', function() {
+                this.deleteMedia();
+            });
 
-        /**
-         * render move media overlay
-         */
-        renderSelectCollection: function() {
-            this.sandbox.start([{
-                name: 'collections/select-overlay@sulumedia',
-                options: {
-                    el: this.$find(constants.moveSelector),
-                    instanceName: 'move-media',
-                    title: this.sandbox.translate('sulu.media.move.overlay-title'),
-                    disableIds: [this.options.data.id]
-                }
-            }]);
-        },
+            // edit media
+            this.sandbox.on('sulu.list-toolbar.edit', function() {
+                this.editMedia();
+            });
 
-        /**
-         * starts overlay for move media
-         */
-        startMoveMediaOverlay: function() {
-            this.sandbox.emit('husky.datagrid.items.get-selected', function(ids) {
-                this.selectedIds = ids;
-                this.sandbox.emit('sulu.media.collection-select.move-media.open');
+            // start collection-select overlay on move-click
+            this.sandbox.on('sulu.list-toolbar.media-move', function() {
+                OverlayManager.
+                    startMoveMediaOverlay(this.sandbox, this.options.id, UserSettingsManager.getMediaLocale());
```

Comment:
@nnatter bad line break. write:
``OverlayManager.startMoveMediaOverlay(
       this.sandbox, this.options.id, UserSettingsManager.getMediaLocale()
  );``